### PR TITLE
#4123 [Manager] Tweak Paper's zoom levels

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4123)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/Toolbar/ToolbarHooks.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ export const useZoom = () => {
 
   const setZoom = (scale, instruction) => {
     const fixedScale = Number((Math.floor(scale * 100) / 100).toFixed(2));
-    const allowedScales = [0.01, 0.03, 0.06, 0.12, 0.25, 0.5, 1.0, 2.0];
+    const allowedScales = [0.25, 0.5, 0.67, 0.75, 0.8, 0.9, 1.0, 1.1, 1.5, 1.5];
```
Comment (eechih) [inline ohara-manager/client/src/components/Pipeline/Toolbar/ToolbarHooks.js]: should be [0.25, 0.5, 0.67, 0.75, 0.8, 0.9, 1.0, 1.1, 1.25, 1.5] :)

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbar/ToolbarHooks.js]: Will do

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
@eechih Any further suggestions before I merge this PR :)

**Comment by konekoya** [comment]
@eechih Thanks for the review :)

**Pull request merged**